src/pages/public_registration/ActionPanel.tsx

```diff
@@ -19,23 +20,61 @@ export const ActionPanel = ({
   isLoadingData,
 }: ActionPanelProps) => {
   const { t } = useTranslation();
+  const [tabValue, setTabValue] = useState(0);
 
   return (
-    <Paper
-      elevation={0}
-      data-testid={dataTestId.registrationLandingPage.tasksPanel.panelRoot}
-      sx={{ gridArea: 'tasks' }}>
-      <StyledPaperHeader>
-        <Typography color="inherit" variant="h1">
-          {t('common.tasks')}
-        </Typography>
-      </StyledPaperHeader>
-      <ActionPanelContent
-        tickets={tickets}
-        refetchData={refetchRegistrationAndTickets}
-        isLoadingData={isLoadingData}
-        registration={registration}
-      />
+    <Paper elevation={0} data-testid={dataTestId.registrationLandingPage.tasksPanel.panelRoot}>
+      <Box sx={{ color: 'primary.contrastText' }}>
+        <Tabs
+          value={tabValue}
+          onChange={(_, newValue) => setTabValue(newValue)}
+          aria-label="action-panel-tabs"
```

Comment:
aria-label er en tekst som leses for brukere med skjermleser, så den burde være en i18n-tekst.
Usikker påå hva den teksten skulle vært her da 🤔 
Mulig det ikke trengs noe, om hver `Tab` har en tekst? (mulig det er helt feil)

Reply:
Skal undersøke 👍